Firestore/Example/Tests/Integration/API/FIRServerTimestampTests.m

```diff
@@ -42,11 +46,20 @@ @implementation FIRServerTimestampTests {
 
   // Listener registration for a listener maintained during the course of the test.
   id<FIRListenerRegistration> _listenerRegistration;
+
+  // Snapshot options that return the previous value for pending server timestamps.
+  FIRSnapshotOptions *_previousValue;
```

Comment:
This name is slightly confusing since we're dealing both in snapshot options and server timestamp values. The name suggests that this is the previous snapshot value, but it's really the option. Something like `_usePreviousOption` could remove this confusion.

Reply:
Reanmed to `_returnPreviousValue` and `_returnEstimatedValue` respectively.